cmd/telegram_bot/bot.go

```diff
@@ -0,0 +1,75 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"github.com/pkg/errors"
+	tele "gopkg.in/telebot.v3"
+	"log"
+	"nodemon/pkg/bots/telegram/config"
+	"nodemon/pkg/bots/telegram/handlers"
+	"nodemon/pkg/bots/telegram/messaging"
+	"os"
+	"os/signal"
+)
+
+func main() {
+	err := run()
+	switch err {
+	case context.Canceled:
+		os.Exit(130)
+	case config.InvalidParameters:
```

Comment:
Looks like not a good place to put the error. Actually this `InvalidParameters` case can be removed, because this in not an interactive command line utility.